corehq/form_processor/tasks.py

```diff
@@ -9,6 +9,10 @@
 
 @no_result_task(queue=SUBMISSION_REPROCESS_CELERY_QUEUE, acks_late=True)
 def reprocess_submission(submssion_stub_id):
-    stub = UnfinishedSubmissionStub.objects.get(id=submssion_stub_id)
```

Comment:
Will do though I think the reason it wouldn't be found is that it was deleted as part of form processing. I changed the order in which the stubs are queued to be according to date created which should also prevent this.